deployment/postgres-init-db/sql/init_merlin.sql

```diff
@@ -94,4 +103,6 @@ begin;
   \ir views/merlin/activity_directive_extended.sql
   \ir views/merlin/simulated_activity.sql
   \ir views/merlin/resource_profile.sql
+  \ir views/merlin/derived_events.sql
+  \ir views/ui/derivation_group_comp.sql -- consider putting this in ui part of schema
```

Comment:
Moving thi to `init_ui.sql` fails as the view depends on a table from `init_merlin.sql` (`merlin.derivation_group`). It only works if left in `init_merlin.sql`.

Reply:
That doesn't make sense. `init_ui.sql` runs after `init_merlin.sql`

Reply:
Can you share the Postgres logs from the failure?

Reply:
Then either pull this line out into `init.sql` with a comment, similar to how we handled two tags tables, or pull out `ui.view` into `init.sql` and reverse `init_ui` and `init_merlin`. The merlin schema init script should not create anything in the ui schema.

Reply:
Additionally, remove the comment on this line, Postgres is trying to read it as code
```
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "--" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "consider" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "putting" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "this" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "in" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "ui" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "part" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "of" ignored
psql:/docker-entrypoint-initdb.d/sql/init_ui.sql:9: warning: \ir: extra argument "schema" ignored
```

Reply:
There isn't much being instantiated in `init_ui.sql`; could we not just reorder `init_ui.sql` to follow `init_merlin.sql` in `init.sql`, instead of extracting individual view and table lines into `init.sql`?

Reply:
That was my "or" suggestion. The `ui.views` table must be instantiated before `init_merlin`, as it is needed to instantiate the `merlin.mission_model` table. Your new UI view depends on tables in the `merlin` schema and must go after that. I don't care which order `init_ui` and `init_merlin` are called, but one of these two tables **must** be pulled out into `init.sql`. 

What follows are your two options for `init.sql`

Option 1:
```sql
  -- UI
  \ir init_ui.sql

  -- Merlin
  \ir init_merlin.sql

  -- Derivation Group Composition View (Object created here due to dependency on Merlin schema)
  \ir views/ui/derivation_group_comp.sql
```

Option 2:
```sql
  -- UI Views Table (Object created here due to dependency in Merlin schema)
  \ir tables/ui/view.sql

  -- Merlin
  \ir init_merlin.sql

  -- UI
  \ir init_ui.sql
```

Reply:
Never mind, rearranging `init_ui.sql` and `init_merlin.sql` caused conflicts with other plans; as such, the suggested approach of extracting just those tables is taken in [extract ui views dependent on merlin to init.sql](https://github.com/NASA-AMMOS/aerie/commit/2be6d82a471953fb96f95ba20d0315963a44ca43).